pkg/workload/tpcc/worker.go

```diff
@@ -200,12 +200,28 @@ func newWorker(
 }
 
 func (w *worker) run(ctx context.Context) error {
+	rng := rand.New(new(rand.LockedSource))
+	rng.Seed(RandomSeed.Seed())
+	// Default tpccTime uses actual time
+	tpccTime := &tpccTime{}
+	if w.config.fakeTime != 0 {
+		timeStep := w.config.fakeTimeStep
+		if timeStep == 0 {
+			timeStep = 10
+		}
+		tpccTime = newTpccTime(
+			timeutil.Unix(int64(w.config.fakeTime), 0),
+			time.Duration(timeStep)*time.Second,
+			RandomSeed.Seed(),
```

Comment:
This means that every worker will have the exact same clock, right? Would be nice to generate a new RNG for each worker. Less relevant for our immediate use-case, which will have a single worker as I understand it.

Reply:
I was thinking that if every worker had their own clock, since it would be seeded with the same seed, then each worker would end up generating the exact same timestamps, which didn't seem as ideal